integration-test/src/main/java/com/sequenceiq/it/cloudbreak/ImageCatalogTests.java

```diff
@@ -0,0 +1,209 @@
+                }), "check imagecatalog is created");
+    }
+
+    @Test(expectedExceptions = BadRequestException.class)
+    public void testCreateInvalidImageCatalogShortName() throws Exception {
+        given(CloudbreakClient.isCreated());
+        given(ImageCatalog.request()
+                .withName(INVALID_IMAGECATALOG_NAME_SHORT)
+                .withUrl(VALID_IMAGECATALOG_URL), "an imagecatalog request with short name"
+        );
+        when(ImageCatalog.post(), "post the imagecatalog request");
+        then(ImageCatalog.assertThis(
+                (imageCatalog, t) -> {
+                    Assert.assertNotEquals(imageCatalog.getResponse().getName(), INVALID_IMAGECATALOG_NAME_SHORT);
+                }),  "check imagecatalog is not created with short name");
+    }
+
+    @Test(expectedExceptions = BadRequestException.class)
+    public void testCreateInvalidImageCatalogLongName() throws Exception {
+        given(CloudbreakClient.isCreated());
+        given(ImageCatalog.request()
+                .withName(longStringGeneratorUtil.stringGenerator(101))
+                .withUrl(VALID_IMAGECATALOG_URL), "an imagecatalog request with long name"
+        );
+        when(ImageCatalog.post(), "post the imagecatalog request");
+        then(ImageCatalog.assertThis(
+                (imageCatalog, t) -> {
+                    Assert.assertNotEquals(imageCatalog.getResponse().getName(), longStringGeneratorUtil.stringGenerator(101));
+                }),  "check imagecatalog is not created with long name");
+    }
+
+    @Test(expectedExceptions = BadRequestException.class)
+    public void testCreateInvalidImageCatalogSpecName() throws Exception {
+        given(CloudbreakClient.isCreated());
+        given(ImageCatalog.request()
+                .withName(SPECIAL_IMAGECATALOG_NAME)
+                .withUrl(VALID_IMAGECATALOG_URL), "an imagecatalog request with special name"
+        );
+        when(ImageCatalog.post(), "post the imagecatalog request");
+        then(ImageCatalog.assertThis(
+                (imageCatalog, t) -> {
+                    Assert.assertNotEquals(imageCatalog.getResponse().getName(), SPECIAL_IMAGECATALOG_NAME);
+                }),  "check imagecatalog is not created with special name");
+    }
+
+    // BUG-97072
+    @Test(expectedExceptions = BadRequestException.class, enabled = false)
+    public void testCreateInvalidImageCatalogUrl() throws Exception {
+        given(CloudbreakClient.isCreated());
+        given(ImageCatalog.request()
+                .withName(VALID_IMAGECATALOG_NAME + "-url")
+                .withUrl("https://" + INVALID_IMAGECATALOG_URL), "an imagecatalog request with invalid url"
+        );
+        when(ImageCatalog.post(), "post the imagecatalog request");
+        then(ImageCatalog.assertThis(
+                (imageCatalog, t) -> {
+                    Assert.assertNotEquals(imageCatalog.getResponse().getName(), VALID_IMAGECATALOG_NAME + "-url");
```

Comment:
You can put this lambda in the method for reuse, it is used in 75th 89th line too